packages/test/src/util/server-tests-runner/index.js

```diff
@@ -72,8 +72,5 @@ exports.run = function(allTests, options) {
     cwd: dir,
     env,
     stdio: "inherit"
-  }).catch(err => {
-    console.error("Error spawning mocha from marko-cli", err);
-    process.exit(1);
-  });
+  }).catch(() => process.exit(1));
```

Comment:
I removed the log because it only rejected if the process exited incorrectly (which is the case if any tests fail). Since we are using `stdio: 'inherit'` all logs are already copied over, this `console.error` as far as I could tell was just a useless stack trace.